pyscript.core/tests/integration/test_02_display.py

```diff
@@ -30,7 +30,7 @@ def test_simple_display(self):
             <script type="py">
                 print('ciao')
                 from pyscript import display
-                display("hello world")
+                display("hello world", append=True)
```

Comment:
I think, if we want to be sure about any default, we can keep one of these without the `append` field ... what do you think?